src/app/users/selectOrganisation.js

```diff
@@ -21,16 +21,28 @@ const getNaturalIdentifiers = async (req) => {
   }
 };
 
+const redirectToSelectOrganisationPage = (req, res, model) => {
```

Comment:
Can we rename it as "render" instead of "redirect" as we are not redirecting the user to another page?

Reply:
renderSelectOrganisationPage

Reply:
done!